lib/xcactivitylog/objects.rb

```diff
@@ -151,10 +151,10 @@ def write_chrome_trace_file(section_type:, to:)
             tid: thread_id,
             ts: section.time_started_recording_usec,
             ph: 'X',
-            name: section.title,
+            name: section.title.dup&.force_encoding('UTF-8'),
```

Comment:
can you add a test for the fixed encoding issue?

Reply:
the new test case will face if this is removed, unless u are talking about a unit test just for this?

Reply:
ah if the new test case fails, then that's sufficient